pkg/utils/util.go

```diff
@@ -376,6 +377,64 @@ func GetSecretReference(secretParams secretParamsMap, snapshotClassParams map[st
 	return ref, nil
 }
 
+// GetSecretReference for the group snapshot
+func GetGroupSnapshotSecretReference(secretParams secretParamsMap, snapshotClassParams map[string]string, snapContentName string, snapshot *crdv1alpha1.VolumeGroupSnapshot) (*v1.SecretReference, error) {
```

Comment:
`snapshotClassParams` to `volumeGroupSnapshotClassParams` , `snapContentName` to `groupSnapshotContentName` and `snapshot` to `volumeGroupSnapshot`